client/site_config_create.go

```diff
@@ -12,117 +12,152 @@ import (
 	"github.com/skupperproject/skupper/api/types"
 )
 
+const (
+	//core options
+	SiteConfigNameKey       string = "name"
+	SiteConfigRouterModeKey string = "router-mode"
+	SiteConfigIngressKey    string = "ingress"
+
+	//console options
+	SiteConfigConsoleKey               string = "console"
+	SiteConfigConsoleAuthenticationKey string = "console-authentication"
+	SiteConfigConsoleUserKey           string = "console-user"
+	SiteConfigConsolePasswordKey       string = "console-password"
+	SiteConfigConsoleIngressKey        string = "console-ingress"
+
+	//router options
+	SiteConfigRouterConsoleKey          string = "router-console"
+	SiteConfigRouterLoggingKey          string = "router-logging"
+	SiteConfigRouterDebugModeKey        string = "router-debug-mode"
+	SiteConfigRouterCpuKey              string = "router-cpu"
+	SiteConfigRouterMemoryKey           string = "router-memory"
+	SiteConfigRouterAffinityKey         string = "router-affinity"
+	SiteConfigRouterAntiAffinityKey     string = "router-anti-affinity"
```

Comment:
One comment on these `affinity` and `anti-affinity` keys. I noticed that most of the
keys match the command line arguments, but the affinity related ones do not, example:
```
Key = router-affinity
Arg = router-pod-affinity
```

and

```
Key = router-anti-affinity
Arg = router-pod-antiaffinity
```

Not sure if they are intended to match each other. What do you think?

Reply:
Agreed, I changed them in the latest update so they match.